### PR TITLE
Add  a popup window

### PR DESCRIPTION
**Pull request by vladanvi99**
In this pull request, we will add a popup window for portfolio projects.

### CODE REVIEW

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @vladanvi99,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [x] Include the project information in the popup window, as it's empty now.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

**Comment by vladanvi99** [comment]
I know that is empty :D
On click, I fill it dynamically, I check key of "see project" button, then I find the item in my array, based on that key.

image

File: js/index.js [hunk 1]
```diff
@@ -9,4 +14,125 @@ window.addEventListener('resize', () => {
...
+
+// POP UP WINDOW FOR PROJECTS
+const portfolioPopUpClose = document.querySelector('.portfolio-window-body .close-portfolio-window-btn button');
+const projectPopUpWrap = document.querySelector('.portfolio-window-wrap');
+const projectPopUpBody = document.querySelector('.portfolio-window-body');
+const seeProjectBtns = [...document.querySelectorAll('.works .work-items .work-item .work-info button')];
+const showProjectPopUpWindow = (btn) => {
+  projectPopUpBody.style.display = 'block';
+  projectPopUpWrap.style.display = 'block';
+  const portfolioProject = myProjects.filter((project) => project.key === btn.dataset.key);
```
Comment (MahmoudBakr23) [inline js/index.js:104]: - [ ] Please do the required changes requested by the previous reviewer and fix the data returned about each project.
_Kindly check the image below and fix the issues regarding the project requirements._
Screenshot

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi, @vladanvi99 and @hiromataba 😃, 

Great work you've done so far on this project. 👏

 ## Highlights 💯

_Well done separating the projects' data in another script file to keep your code clean._

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 1 above)
Comment (vladanvi99) [reply js/index.js]: Sorry for last my last comment my linter change "==" to "===", now I just convert it in number
image

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi, @vladanvi99 and @hiromataba 😃, 

Great work you've done so far on this project. 👏

 ## Highlights 💯

_Well done making the required changes. However, there is still one more small issue to make in order to get approval._

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

Kindly fix the issue with the lists right there.
Screenshot

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi, @vladanvi99 and @hiromataba 😃, 

Well done 💯 🥇 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**